app/workers/evss/disability_compensation_form/submit_uploads.rb

```diff
@@ -17,24 +17,36 @@ def self.start(uuid)
         batch.jobs do
           claim_id = get_claim_id(uuid)
           uploads = get_uploads(uuid)
-          uploads.each { |u| perform_async(uuid, u[:guid], claim_id) }
+          uploads.each_with_index do |upload_data, _index|
```

Comment:
the `each_with_index` was for logging the index, for now change it back to `each` and we'll change back again when needed